### PR TITLE
2.3.1 Update changelog

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,15 +8,21 @@
...
+* Tango skin: Show crossfader assign buttons by default [#4046](https://github.com/mixxxdj/mixxx/pull/4046)
+* Fix keyfinder library in arm64 builds [#4047](https://github.com/mixxxdj/mixxx/pull/4047)
+* Fix wrong track being recorded in History [lp1933991](https://bugs.launchpad.net/mixxx/+bug/1933991) [#4041](https://github.com/mixxxdj/mixxx/pull/4041) [#4059](https://github.com/mixxxdj/mixxx/pull/4059) [#4107](https://github.com/mixxxdj/mixxx/pull/4107)
 * Fix support for relative paths in the skin system which caused missing images in third-party skins [#4151](https://github.com/mixxxdj/mixxx/pull/4151)
 * Fix relocation of directories with special/reserved characters in path name [#4146](https://github.com/mixxxdj/mixxx/pull/4146)
+* Update keyboard shortcuts sheet [#4042](https://github.com/mixxxdj/mixxx/pull/4042)
+* Library: resize the Played checkbox and BPM lock with the library font [#4050](https://github.com/mixxxdj/mixxx/pull/4050)
+* Don't allow Input focus on waveforms [#4134](https://github.com/mixxxdj/mixxx/pull/4122)
```
Comment (Swiftb0y) [inline CHANGELOG.md:18]:
```suggestion
* Don't allow Input focus on waveforms [#4134](https://github.com/mixxxdj/mixxx/pull/4134)
```

**Review by Swiftb0y — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,15 +8,21 @@
...
+* Library: resize the Played checkbox and BPM lock with the library font [#4050](https://github.com/mixxxdj/mixxx/pull/4050)
+* Don't allow Input focus on waveforms [#4134](https://github.com/mixxxdj/mixxx/pull/4134)
 * Fix performance issue on AArch64 by enabling flush-to-zero for floating-point arithmetic [#4144](https://github.com/mixxxdj/mixxx/pull/4144)
 * Fix custom key notation not restored correctly after restart [#4136](https://github.com/mixxxdj/mixxx/pull/4136)
+* Traktor S3: Disable scratch when switching decks to prevent locked scratch issue [#4073](https://github.com/mixxxdj/mixxx/pull/4073)
 
 ### Packaging
 
-* Install udev rule file to the correct location automatically in the CMake install step when CMAKE_INSTALL_PREFIX is `/` or `/usr`. It is no longer necessary to manually copy the udev rule file in packaging scripts. [#4126](https://github.com/mixxxdj/mixxx/pull/4126)
+* Use pkg-config udevdir, to install the rules file the CMake install step when CMAKE_INSTALL_PREFIX is `/` or `/usr`. It is no longer necessary to manually copy the udev rule file in packaging scripts. [#4126](https://github.com/mixxxdj/mixxx/pull/4126)
```
Comment (uklotzde) [inline CHANGELOG.md:25]: The first sentence doesn't seem to be valid. I also don't understand this paragraph, i.e. what I am supposed to do as a package maintainer. Do I need to execute `pkg-config udevdir`?? That doesn't seem to do anaything.

**Review by uklotzde — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (daschuer) [reply CHANGELOG.md:25]: Ah ok, that is from the commit message imperative. I need to rephrase it.

Comment (daschuer) [reply CHANGELOG.md]: Hopefully better now.

**Comment by Be-ing** [comment]
This was merged to main, not 2.3.

**Comment by Holzhaus** [comment]
> This was merged to main, not 2.3.

This wasn't merged at all.

**Comment by Be-ing** [comment]
Oh my bad :laughing:

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,16 +8,24 @@
...
-* Fix various build issues on FreeBSD [#4122](https://github.com/mixxxdj/mixxx/pull/4122) [#4123](https://github.com/mixxxdj/mixxx/pull/4123) [#4124](https://github.com/mixxxdj/mixxx/pull/4124)
+* It is no longer necessary to manually copy the udev rule file in packaging scripts. Now pkg-config is used to determine the udevdir used to install the rules file in the CMake install step when CMAKE_INSTALL_PREFIX is `/` or `/usr`.  [#4126](https://github.com/mixxxdj/mixxx/pull/4126)
+* Various build issues on FreeBSD are fixed [#4122](https://github.com/mixxxdj/mixxx/pull/4122) [#4123](https://github.com/mixxxdj/mixxx/pull/4123) [#4124](https://github.com/mixxxdj/mixxx/pull/4124)
+* .desktop file has be renamed to org.mixxx.Mixxx.desktop according to Freedesktop standards [#4206](https://github.com/mixxxdj/mixxx/pull/4206)
+* Uses system provided hidapi library if Version >= 0.10.1 [#4215](https://github.com/mixxxdj/mixxx/pull/4215)
```
Comment (Be-ing) [inline CHANGELOG.md:28]:
```suggestion
* Uses system provided hidapi library if version >= 0.10.1 [#4215](https://github.com/mixxxdj/mixxx/pull/4215)
```

Comment (Be-ing) [inline CHANGELOG.md:28]: Please add:
```
* Please update PortAudio to [19.7](https://github.com/PortAudio/portaudio/releases/tag/v19.7.0) if you have not done so already. This is required for Mixxx to work with PipeWire via the JACK API for many devices.
```

**Review by uklotzde — APPROVED** [review]
The revised section about pkgconfig and udev is clear to me now.

Ready after fixing the remaining review comments from @Be-ing.

**Comment by daschuer** [comment]
Done.

**Review by Be-ing — APPROVED** [review]

**Pull request merged**